bin/strings-import.js

```diff
@@ -2,74 +2,78 @@
 "use strict";
 
 /* eslint-disable no-console */
+const fetch = require("node-fetch");
+
+/* globals cd, ls, mkdir, rm, ShellString */
+require("shelljs/global");
 
 const DEFAULT_LOCALE = "en-US";
+const L10N_CENTRAL = "https://hg.mozilla.org/l10n-central";
+const PROPERTIES_PATH = "raw-file/tip/browser/extensions/activity-stream/activity-stream.properties";
+const STRINGS_FILE = "strings.properties";
 
-/* globals cd, cp, ls, mkdir, rm */
-require("shelljs/global");
+// Get all the locales in l10n-central
+async function getLocales() {
+  console.log(`Getting locales from ${L10N_CENTRAL}`);
 
-const args = process.argv.slice(2);
-if (args.length < 1) {
-  throw Error("Please provide the path to the target strings repository");
+  const html = await (await fetch(L10N_CENTRAL)).text();
```

Comment:
You can use `https://hg.mozilla.org/l10n-central/?style=raw`

Reply:
... or json, depending on your preference.

Reply:
I didn't know about https://hg.mozilla.org/l10n-central/?style=json

That makes things a lot less hacky